service/surf/apps/filters/utils.py

```diff
@@ -81,18 +83,18 @@ def _update_mptt_filter_category(filter_category, api_client):
         if created or filter_item.title_translations is None:
             is_new = True
 
-            if filter_category.external_id in AUTO_ENABLED_FILTERS:
-                _auto_enable_mptt_filter_item(filter_item)
+            if filter_category.external_id in ENGLISH_SAME_AS_DUTCH:
+                _auto_enable_english_same_as_dutch(filter_item)
             else:
                 _translate_mptt_filter_item(filter_item)
 
         yield external_id, is_new
 
 
-def _auto_enable_mptt_filter_item(filter_item):
+def _auto_enable_english_same_as_dutch(filter_item):
     translation = Locale.objects.create(
         asset=f"{filter_item.external_id}_auto_generated_at_{datetime.datetime.now().strftime('%c-%f')}",
-        en=filter_item.external_id, nl=filter_item.external_id, is_fuzzy=True
+        en=filter_item.external_id, nl=filter_item.external_id, is_fuzzy=False
```

Comment:
Dit is nog steeds fuzzy. Zolang het automatisch gaat zal er iemand een blik op moeten werpen.

Reply:
Dit zijn schrijvers en publishers. Dat zijn allemaal namen. Daarom had ik het weggehaald

Reply:
Het is niet helemaal waar alleen. Universiteit Utrecht is in Engels Utrecht University.